docs/topics/media-pipeline.rst

```diff
@@ -127,9 +128,59 @@ Whose ``SHA1 hash`` is::
 
     3afec3b4765f8f0a07b78f98c07b83f013567a0a
 
-Will be downloaded and stored in the following file::
+Will be downloaded and stored in your chosen :ref:`storage method <topics-supported-storage>` using the following file name::
+
+   3afec3b4765f8f0a07b78f98c07b83f013567a0a.jpg
+
+Custom File Naming
+-------------------
+You may wish to use a different calculated file name for saved files.
+For example, classifying an image by including meta in the file name.
+
+Customize file names by overriding the ``file_path`` class method of your
+pipeline.
```

Comment:
“class method” may be mistaken for `@classmethod`, so just “method” would be best.

Also, “pipeline” is too generic (could be mistaken for “item pipeline”), “media pipeline” would be best.